common/util/autoware_lint_common/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>autoware_lint_common</name>
+  <version>0.1.0</version>
+  <description>The list of commonly used linters in the ament buildsytem in CMake.</description>
```

Comment:
TODO: Refer to ament_lint_auto in the description.

TODO: Send a PR to upstream?
https://github.com/ament/ament_lint/blob/ebd524bb9973d5ec1dc48a670ce54f958a5a0243/ament_lint_common/package.xml#L6